images.md

```diff
@@ -0,0 +1,22 @@
+# Lambda Instance images creation
```

Comment:
Isn't there an ansible script that is supposed to be usde for configuring the system before taking the image? Is this described somewhere?